RedHat-UX/red-hat-design-system
# docs: clean up content at various pages (e.g., 404)

## What I did

1. Updated the page title and heading structure of the 404 page.
2. Added a link to the Orca screen reader section
3. Changed "Content" heading text for Card's Guidelines page.


## Testing Instructions

1. Go to the [DP 404 page](https://deploy-preview-1999--red-hat-design-system.netlify.app/404page) and verify the following
  - The page title is now `<title>Page not found (404 error) | Red Hat design system</title>` and not `<title>Oops! | Red Hat design system</title>`
  - The 404 page only has one (_and only one_) `<h1>` element: "Page not found (404 error)"

2. Go to the [DP Orca screen reader section](https://deploy-preview-1999--red-hat-design-system.netlify.app/accessibility/screen-readers-computer/#orca-(linux)) and verify that there is a "Keyboard shortcuts" link in the unordered list that points to https://a11ysupport.io/learn/at/orca

3. Go to the [DP Card > Guidelines page](https://deploy-preview-1999--red-hat-design-system.netlify.app/elements/card/guidelines/#content-cards) and verify the following:
  - The former "Content" heading now reads "Content cards"
  - The `font-family` is Red Hat Display, and no longer monospaced
